transformer/src/main/scala/com/snowplowanalytics/snowflake/transformer/TransformerJob.scala

```diff
@@ -52,13 +52,33 @@ object TransformerJob {
     sc.register(keysAggregator)
 
     val events = sc
-      .textFile(jobConfig.input)
-      .map { e => Transformer.jsonify(e) }
+      .wholeTextFiles(jobConfig.input)
+      .flatMap(rdd => {
```

Comment:
```suggestion
      .flatMap(pair => {
```
I believe the argument of `flatMap` is of type `Tuple2<String,String>`, isn't it? So, naming it `rdd` might be a bit misleading.

Reply:
I verified that in this scope we are passed an `RDD<String, String>` so I'll keep it as it is 👍